go/proto/api/v1/service_visibility.pb.yarpc.go

```diff
@@ -261,14 +261,20 @@ func NewFxVisibilityAPIYARPCProcedures() interface{} {
 				Server:      params.Server,
 				AnyResolver: params.AnyResolver,
 			}),
-			ReflectionMeta: reflection.ServerMeta{
-				ServiceName:     "uber.cadence.api.v1.VisibilityAPI",
-				FileDescriptors: yarpcFileDescriptorClosurea7341dc69cef4364,
-			},
+			ReflectionMeta: VisibilityAPIReflectionMeta,
```

Comment:
not concerning since it passed CI, just curiosity: any idea why this changed?
sorta smells like it might be a yarpc-code-generator version change or something, unless the mere existence of fields changes this behavior for some reason.

Reply:
No idea, honestly didn't look at the generated files, but I think it's a good guess that the existence of fields requires some extra code.
The empty errors has a message by default, so would make sense that adding a field would include some reflection?
But all speculation

Reply:
So I just checked on master, and it also generates this change, same for Jan. We tried to figure out why, but everything looks versioned, so it's strange? I agree it's not a concern as it's effectively the same code, and as you say, the CI parses.